tests/test_external_converter.py

```diff
@@ -63,7 +63,8 @@ def custom_converter(params):
     external_opset_imports = {}
     if domain is not None:
         external_opset_imports[domain] = version
-        expected_warning_num += 1
+        # TODO(take-cheeze) Investigate why onnx.checker.check_model succeed
+        # expected_warning_num += 1
```

Comment:
ref: https://github.com/onnx/onnx/pull/1876 , from onnx1.5, model check does not occur error when domain is set. onnx-chainer should not be cause warning to fit it. But TODO is OK, and fix it later.